### PR TITLE
Add Travis CI (React Frontend)

### PR DESCRIPTION
**Pull request by JosephSemrai**
# Description

Added Travis configuration for React application.
- Fixed 'renders without crashing' test as it would crash since it was not given a provider previously.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Ran Travis locally.
- [x] Test A

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -22,22 +22,22 @@ jobs:
...
+        - "stable"
+      cache:
+        directories:
+          - node_modules
+      before_script: cd labellab-client
+      script:
+        - npm install
+        - npm test
+        - npm run build
+      deploy:
```
Comment (Tushar19varshney) [inline .travis.yml]: You can remove deploy script as we don't need that now.

**Comment by JosephSemrai** [comment]
The build continues to fail due to a dependency mentioned in the package.json that is already a dependency of create-react-app. Should I address this in this pull request?

**Comment by ivantha** [comment]
> The build continues to fail due to a dependency mentioned in the package.json that is already a dependency of create-react-app. Should I address this in this pull request?

Please do 🙂

**Comment by UdeshUK** [comment]
Closing because of conflicts and inactivity

**Pull request closed without merging**